### DISCUSSION TITLE
Woche 15.01-22.01

### DISCUSSION THREAD

**Discussion by carolina-cza** [carolina-cza/Tecna#7] [Weekly Blog]
Moin.
Die Woche unter anderem bisschen mit chatbots in Python auseinander gesetzt und hab eine ganz coole Seite zu dem Thema gefunden, wo die basics step for step erklärt werden [https://realpython.com/build-a-chatbot-python-chatterbot/](url). Fand ich sehr interessant und ich bin am überlegen ob ich es zu einem späteren mal selbst ausprobiere, weil ich könnte mir echt vorstellen das es ganz lustig sein könnte sowas zu bauen. Aber vorher will ich noch meine Basis Wissen in Python erweitern deswegen hab ich mich noch mal mit dictionary beschäftigt und mit password checker [https://github.com/carolina-cza/Tecna/blob/codespace-super-space-yodel-x55ppqjx97jqfp699/daily_stuff/checkpassword.py](url) und da hab ich mir auch paar verschieden Tutorials angeschaut. 
Allgemein kann ich sagen das ich mich die Woche relativ viel mit codes beschäftigt hab, für meine Verhältnisse ^^

**Comment by Roadraudi** [comment]
Moin! Es klingt so, als ob du eine produktive und interessante Woche hattest. Die Auseinandersetzung mit Chatbots in Python ist definitiv ein faszinierendes Thema. Der Gedanke, selbst einen Chatbot zu bauen, klingt nach einer spannenden Herausforderung und einer Möglichkeit, deine Python-Fähigkeiten weiter zu vertiefen.

**Comment by DGrillCoder** [comment]
Hört sich spannend an. Weiter so
